qiskit_experiments/library/characterization/__init__.py

```diff
@@ -48,20 +48,34 @@
     :toctree: ../stubs/
     :template: autosummary/analysis.rst
 
-    T1Analysis
-    T2RamseyAnalysis
-    CrossResonanceHamiltonianAnalysis
-
+    ~analysis.T1Analysis
```

Comment:
```suggestion
    analysis.T1Analysis
```
I don't think tilde is necessary, can you check docs? (this prevents to generate docs twice, but there is no other place generating docs for these analysis)

Reply:
Done here: c13d4eb. Docs look fine.